### DISCUSSION TITLE
Next webauthn4j-spring-security release

### DISCUSSION THREAD

**Discussion by TGITS** [webauthn4j/webauthn4j-spring-security#1166] [General]
Hello,

First, thanks to @ynojima and all the contributors for this project and all the work done.

I have been using in a small proof-of-concept projet **webauthn4j** and **webauthn4j-spring-security**.

However, in the context of using **webauthn4j** and **webauthn4j-spring-security** for a project that will be more than a mere POC, I would like to know whether a new release  of **webauthn4j-spring-security** is planned in the near  future.

As a matter of fact, the latest release of **webautn4j** is the 0.21.0.RELEASE in date of the 1st of March 2023 but the latest release of **webauthn4j-spring-security** is the 0.7.6.RELEASE in date of the 16th of August 2022. This release uses the 0.20.2.RELEASE of webautn4j.

To validate the use of **webauthn4j** and **webauthn4j-spring-security** for my project, I would need a little visibility on the planning of releases.

**Comment by ynojima** [comment]
`webauthn4j` is released monthly basis. but `webauthn4j-spring-security` is released when it has appropriate updates. 
I will release the next webauthn4j-spring security version within a month.

**Reply by TGITS** [reply]
Thank you @ynojima for your answer !

**Reply by ynojima** [reply]
[0.8.0.RELEASE](https://github.com/webauthn4j/webauthn4j-spring-security/releases/tag/0.8.0.RELEASE), which targets Spring Security 5.8 is released.
[0.9.0.RELEASE](https://github.com/webauthn4j/webauthn4j-spring-security/releases/tag/0.9.0.RELEASE), which targets Spring Security 6.0 is released.
